### ISSUE TITLE
Region Issues and Inconsistancies

### ISSUE THREAD

**Issue by terracoda** [phetsims/ratio-and-proportion#237]
Just listing things I find as I work on the refining the design for the Object and Context Responses for all the hand interactions.

Region Issues

1. Sliders - locked ratio - left hand in middle region, right at top
- left should be at middle
- wondering if this is related to snapping.
- this issue might be resolved by issue #245 and #241

2. Both Hands, Ratio Locked, hands near bottom (0.5 and 1), the description says "very close to each other, both hands in lower region"
- I think the hands are "almost even", in lower region
- What is calculating "very close to each other" that regions seems too wide.
- This issue should be resolved with #242

3. Currently, when using Lock Ratio, I get slightly different responses depending on whether my focus in on a slider or on Both Hands.
- When on a slider, I get "closer together/farther apart" on directional changes
- The Both Hands interaction doesn't have progress changes built in, hence them not being there.
- When ratio is locked, we'll want both interactions to be using the same strings.
- This issue will likely be resolved as we work through #245
